<launch>
    <node name="depth_viewer" pkg="orb_slam3_ros_wrapper" type="orb_slam3_ros_wrapper_depth_viewer" output="screen">
        <remap from="/camera/aligned_depth_to_color/image_raw" to="/camera/aligned_depth_to_color/image_raw"/>
    </node>
</launch>
